Makefile

```diff
@@ -12,6 +12,23 @@ all:
 		--exclude-path proto/asgt/v2/type \
 		--exclude-path proto/ssn/type \
 		--exclude-path proto/gen_bq_schema \
+		--exclude-path proto/ssn/dataservice \
+		--include-imports \
+		--verbose
+
+#	grpc C# plugin generates all files to root folder
+#       this results in ssn/dataservice and asgt/dataservice being generated under the same filename "DataserviceGrpc.cs" which will overwrite whichever gets generated first
+#	this workaround generates asgt/dataservice first, renames the file and then generates the other one (see exclude paths above and below)
```

Comment:
I still don't like this but I also don't any better better way... So I'll just leave this comment here as a rant into the void 😄